trunk/SUAVE/Plots/Geometry/plot_vehicle.py

```diff
@@ -360,28 +361,22 @@ def generate_nacelle_points(nac,tessellation = 24):
     
     num_nac_segs = len(nac.Segments.keys())   
     theta        = np.linspace(0,2*np.pi,tessellation)
-    n_points     = 20
+    n_points     = 21 
```

Comment:
this seems like a number that should be defaulted

Reply:
corrected